src/components/banner.jsx

```diff
@@ -15,7 +19,7 @@ function Banner({ children, languageList, slug }) {
           <Grid row gap className="banner__logo-title">
             <Grid>
               <div>
-                <img src={logoBanner} alt={t('location')} />
+                {logoBanner && <img src={logoBanner} alt={t('location')} />}
```

Comment:
You can still have a banner without the logo. Although it is highly encouraged to have a logo, what we have here might be confusing for a programmer trying to figure out why there is no banner. Instead, if the banner exists but the logo doesn't, they can assume it's because they did not configure the logo properly.

Reply:
This shouldn't prevent the banner from rendering, only the image of the logo if there is no src